packages/ui/src/composables/useHTMLElement.ts

```diff
@@ -0,0 +1,36 @@
+import { computed, getCurrentInstance, onBeforeUnmount, onMounted, onUpdated, Ref, ref } from 'vue'
+
+const extractHTMLElement = (el: any): HTMLElement => el && '$el' in el ? el.$el : el
```

Comment:
This (extracting $el) is used at least in 4 composables for now. May be we should move it to global helpers?

P.S. Isn't `=> el?.$el : el` better?